django/contrib/admin/templates/admin/auth/user/unusable_password_warning.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+<template id="id_unusable_warning_template">
+  <ul id="id_unusable_warning" class="messagelist">
+    <li class="warning">
+      {% blocktranslate %}
+        Not setting a password will render the login credentials unusable.
+        The user will not be able to authenticate using a password until a
+        new password is set. Other authentication mechanisms, such as
+        Single Sign-On or LDAP, remain unaffected.
```

Comment:
I think the first sentence just adds confusion so should definitely be removed. Starting with "not" makes it tough to read, and "login credentials" will be lost on people. So at least, do this:

```suggestion
        The user will not be able to authenticate using a password until a
        new password is set. Other authentication mechanisms, such as
        Single Sign-On or LDAP, remain unaffected.
```

I’d go further and use even simpler language. Don’t care too much about the exact wording but the specific changes:

- "authenticate" / "authentication mechanism" -> "log in"
- "remain unaffected" -> needs to use simpler language too.

Or even simpler:

```suggestion
        The user will not be able to log in with a password.
        They may still be able to log in through other options, such as Single Sign-On or LDAP.
```

Reply:
@thibaudcolas the thing is that "log in" is not (strictly) [the same](https://english.stackexchange.com/questions/80862/difference-between-log-in-authenticate-and-sign-in) as "authenticate". One could authenticate via other means (think about [Basic Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication)) and still not do a "log in" flow.

This new UI affects how authentication works, not necessarily the login process.